Makefile

```
@@ -1,112 +1,296 @@
+# |          |              | LLVM       | JIT         | glibc      |         no |
+# |          |              |            | Native      | glibc      |        yes |
+# |          |              |            |             |            |            |
+# |          |              | Singlepass | JIT         | glibc      |         no |
+# |          |              |            | Native      | glibc      |         no |
+#-|----------|--------------|------------|-------------|------------|------------|
+# | Darwin   | amd64        | Cranelift  | JIT         | glibc      |        yes |
+# |          |              |            | Native      | glibc      |        yes |
+# |          |              |            |             |            |            |
+# |          |              | LLVM       | JIT         | glibc      |        yes |
+# |          |              |            | Native      | glibc      |        yes |
+# |          |              |            |             |            |            |
+# |          |              | Singlepass | JIT         | glibc      |        yes |
+# |          |              |            | Native      | glibc      |         no |
+# |          |              |            |             |            |            |
+# |          | aarch64      | Cranelift  | JIT         | glibc      |        yes |
+# |          |              |            | Native      | glibc      |         no |
+# |          |              |            |             |            |            |
+# |          |              | LLVM       | JIT         | glibc      |         no |
+# |          |              |            | Native      | glibc      |        yes |
+# |          |              |            |             |            |            |
+# |          |              | Singlepass | JIT         | glibc      |         no |
+# |          |              |            | Native      | glibc      |         no |
+#-|----------|--------------|------------|-------------|------------|------------|
+# | Windows  | amd64        | Cranelift  | JIT         | glibc      |        yes |
+# |          |              |            | Native      | glibc      |         no |
+# |          |              |            |             |            |            |
+# |          |              | LLVM       | JIT         | glibc      |         no |
+# |          |              |            | Native      | glibc      |         no |
+# |          |              |            |             |            |            |
+# |          |              | Singlepass | JIT         | glibc      |         no |
+# |          |              |            | Native      | glibc      |         no |
+# |----------|--------------|------------|-------------|------------|------------|
+
+
+#####
+#
+# Define the “Platform” and “Architecture” columns of the matrix.
+#
+#####
+
+
+IS_DARWIN := 0
+IS_LINUX := 0
+IS_WINDOWS := 0
+IS_AMD64 := 0
+IS_AARCH64 := 0
+LIBC ?=
```

```suggestion
LIBC :=
```

Now that #2003 changed the build process for `musl` we can safely change the operator here. The only thing I'm wondering is will it break the pipeline in your PR? I don't know.

So two choices here:
* you change it in your PR
* I rebase #2003 on top of your patch and then do this change on my side.